experimental/packages/opentelemetry-exporter-prometheus/src/PrometheusSerializer.ts

```diff
@@ -250,19 +266,47 @@ export class PrometheusSerializer {
       ? `\n# UNIT ${name} ${escapeString(metricData.descriptor.unit)}`
       : '';
     const type = `# TYPE ${name} ${toPrometheusType(metricData)}`;
+    let additionalAttributes: Attributes | undefined;
+
+    if (this._withoutScopeInfo) {
+      additionalAttributes = this._additionalAttributes;
+    } else {
+      additionalAttributes = Object.assign(
+        {
+          [ATTR_OTEL_SCOPE_NAME]: scope.name,
+          [ATTR_OTEL_SCOPE_SCHEMA_URL]: scope.schemaUrl ?? '',
+          [ATTR_OTEL_SCOPE_VERSION]: scope.version ?? '',
```

Comment:
I'm not too knowledgeable about prometheus - so what I'm saying might not be correct/applicable here:

Q: is it okay to have this default to the empty string even when it's undefined? 🤔 

Why I'm asking: I know in some other metrics backends, an empty string can be different from an attribute not being present at all - so adding a `schemaUrl` could technically be a breaking change, since the data changes from `''` to, let's say, `https://example.com/schema.json`. Any query that relies on the empty string being there breaks when the value changes.

However, when the attribute/label is not present, then it can be added by telemetry later: since nobody could depend on it in their queries earlier, it's not considered a breaking change.